src/backends/perf_pt/mod.rs

```diff
@@ -616,6 +612,22 @@ mod tests {
         (ptxed_args, vdso_tempfile)
     }
 
+    /*
+     * Determine if the given x86_64 assembler mnemonic should terminate a block.
+     *
+     * Mnemonic assumed to be lower case.
```

Comment:
Why?

Reply:
Why is it assumed lower case? `ptxed` prints instructions in lower case and that's where the input comes from.

I did consider explicitly lower-casing, but this will slow the tests down, especially the long one.

Note that this is testing-only code. In the main library, we use integer comparisons. We can't use them here because we are scraping the output of a command.

Reply:
Maybe just add an `assert` to check for lower case rather than documenting it?

Reply:
Are strings counted in Rust? Otherwise we are still going to slow tests down.

Reply:
Counted?

Reply:
Wait, that's rubbish, what I've just said. It doesn't matter, the assertion will force a linear pass over each line of assembler code.

Reply:
Oddly enough, adding
```
assert_eq!(instr.find(|c: char| !c.is_lowercase()), None);
```

Doesn't slow the long benchmark down. Would that work for you?